quickwit/quickwit-search/src/lib.rs

```diff
@@ -105,6 +112,40 @@ impl GlobalDocAddress {
     }
 }
 
+impl std::fmt::Display for GlobalDocAddress {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        f.write_str(&self.split)?;
+        write!(
+            f,
+            ":{:08x}:{:08x}",
+            self.doc_addr.segment_ord, self.doc_addr.doc_id
+        )
+    }
+}
+
+impl std::str::FromStr for GlobalDocAddress {
```

Comment:
do we use this?

Reply:
yes, when we parse `search_after` for a doc id